CMakeLists.txt

```diff
@@ -209,7 +209,7 @@ macro(commom_lib_settings lib write )
     PUBLIC
         $<BUILD_INTERFACE:${CMAKE_CURRENT_SOURCE_DIR}/include>
         $<INSTALL_INTERFACE:include>
-
+    PRIVATE
         $<BUILD_INTERFACE:${CMAKE_CURRENT_SOURCE_DIR}/lib/basisu/transcoder>
         $<INSTALL_INTERFACE:lib/basisu/transcoder>
```

Comment:
Is this line really necessary? `lib/basisu/transcoder` is never installed.

Reply:
Yes, it is necessary. Otherwise the transcoder headers are defined as being part of the public interface of the ktx library target. As they are however not installed, this results in an error when cmake `find_package(Ktx)` validates the include directories of the imported target.